src/postgres_mcp/server.py

```diff
@@ -198,6 +199,23 @@ async def analyze_single_query(
     return format_text_response(result)
 
 
+@mcp.tool(
+    description="Analyzes database health for specified components including buffer cache hit rates, "
+    "identifies duplicate, unused, or invalid indexes, sequence health, constraint health "
+    "vacuum health, and connection health."
+)
+async def database_health(health_type: str = "all") -> ResponseType:
+    """Analyze database health for specified components.
+
+    Args:
```

Comment:
The valid forms for `health_type` are in the comment but should be in the description.

Reply:
added